class-00/lab-c/starter-code/index.html

```diff
@@ -6,6 +6,9 @@
   <title>Choco Pizza</title>
   <link rel="stylesheet" href="styles/reset.css">
   <link rel="stylesheet" href="styles/style.css">

```

Comment:
You no longer have a style.css file. This line should be removed so we don't waste time looking for a broken link.